TAF/utils/scripts/docker/get-compose-file-perfermance.sh

```diff
@@ -10,10 +10,13 @@ USE_SECURITY=${2:--}
 [ "$USE_SECURITY" != '-security-' ] && USE_NO_SECURITY="-no-secty"
 
 # # sync docker-compose file from developer-script repo
-./sync-nightly-build.sh master ${USE_NO_SECURITY} ${USE_ARM64}
+./sync-nightly-build.sh master "${USE_NO_SECURITY}" "${USE_ARM64}" "-perf"
 
-cp docker-compose-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose.yaml
+cp docker-compose-taf-perf-nexus${USE_NO_SECURITY}${USE_ARM64}.yml docker-compose-mqtt.yaml
 
-# Insert required services for retrieving exported time
-sed -e '1,/- edgex-network/d' docker-compose-end-to-end.yaml > docker-compose-end-mqtt.yaml
-sed -e '/app-service-rules:/r docker-compose-end-mqtt.yaml' -e //N docker-compose-nexus${USE_NO_SECURITY}${USE_ARM64}.yml > docker-compose-mqtt.yaml
+sed -i '/PROFILE_VOLUME_PLACE_HOLDER: {}/d' docker-compose-mqtt.yaml
+sed -i 's/\CONF_DIR_PLACE_HOLDER/${CONF_DIR}/g' docker-compose-mqtt.yaml
+sed -i 's/\PROFILE_VOLUME_PLACE_HOLDER/${WORK_DIR}\/TAF\/config\/${PROFILE}/g' docker-compose-mqtt.yaml
+sed -i 's/\EXPORT_HOST_PLACE_HOLDER/${DOCKER_HOST_IP}/g' docker-compose-mqtt.yaml
```

Comment:
These 4 lines are unnecessary on the script.

Reply:
@cherrycl , correct for the first 3, but `EXPORT_HOST_PLACE_HOLDER` is needed.

Reply:
unneeded lines have been removed

Reply:
@lenny-intel Sorry, I forgot to submit the comment for http-export service doesn't use on performance metrics test on the [PR #373](https://github.com/edgexfoundry/developer-scripts/pull/373).

If the app-service-http-export is not in the docker-compose-mqtt.yaml file, the `EXPORT_HOST_PLACE_HOLDER` is unnecessary.